api-samples/printing/printers.js

```diff
@@ -83,25 +89,49 @@ function createPrintersTable() {
         }
 
         const printTd = document.createElement('td');
-        printTd.appendChild(
-          createPrintButton(function () {
-            onPrintButtonClicked(
-              printer.id,
-              response.capabilities.printer.dpi.option[0]
-            );
-          })
-        );
+        printTd.appendChild(createButton('Print', function() {
+          onPrintButtonClicked(
+              printer.id, printerInfo.capabilities.printer.dpi.option[0]);
+        }));
         tr.appendChild(printTd);
 
         tbody.appendChild(tr);
       });
-    }
-
+    });
     const table = document.getElementById('printersTable');
     table.appendChild(tbody);
   });
+  chrome.printing.onJobStatusChanged.addListener((jobId, status) => {
+    console.log("jobId: " + jobId + ", status: " + status);
+    let jobTr = document.getElementById(jobId);
+    if (jobTr == undefined) {
+      jobTr = document.createElement("tr");
+      jobTr.setAttribute("id", jobId);
+      const jobIdTd = document.createElement('td');
+      jobIdTd.appendChild(document.createTextNode(jobId));
+      jobTr.appendChild(jobIdTd);
+      let jobStatusTd = document.createElement('td');
+      jobStatusTd.setAttribute("id", jobId + "-status");
+      jobStatusTd.appendChild(document.createTextNode(status));
+      jobTr.appendChild(jobStatusTd);
+
+      const cancelTd = document.createElement('td');
+      let cancelBtn = createButton('Cancel', function() {
+        onCancelButtonClicked(jobId);
+      })
```

Comment:
@oliverdunk I may change this yet again. I know this kind of thing is what you do in a working application. But it makes it really hard to figure out what's going on in a sample.